priam/src/main/java/com/netflix/priam/backup/AbstractBackupPath.java

```diff
@@ -147,15 +146,13 @@ protected String match(Date start, Date end) {
     /** Local restore file */
     public File newRestoreFile() {
         File return_;
-        String dataDir = config.getDataFileLocation();
+        String dataDir = config.getRestoreDataLocation();
         switch (type) {
-            case CL:
-                return_ = new File(PATH_JOINER.join(config.getBackupCommitLogLocation(), fileName));
-                break;
             case SECONDARY_INDEX_V2:
-                String restoreFileName =
-                        PATH_JOINER.join(dataDir, keyspace, columnFamily, indexDir, fileName);
-                return_ = new File(restoreFileName);
+                return_ =
```

Comment:
While we are in here, consider renaming this variable, its not idiomatic java

Reply:
Done.